apps/dashboard/components/EmailInboxSettingsTab.tsx

```diff
@@ -75,13 +77,13 @@ function EmailInboxSettingsTab({ inboxId }: Props) {
                   }
                 },
                 color: 'danger',
-                children: 'Delete',
+                children: `${t('deleteCta')}`,
```

Comment:
should not be structured this way. ctas should be a nested object where relevant.